using-timescaledb/continuous-aggregates.md

```diff
@@ -18,118 +18,127 @@ re-calculate all of the data on every refresh. Only new and/or
-not provide one, you will get an error. The `time_bucket` function in
-this case has a bucket width of 1 hour.
+Lastly, you should add a policy to ensure that the continuous
+aggregate is refreshed on a regular basis.
+
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary_daily',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '1 h');
+```
+
+In this case, the continuous aggregate will be refreshed every hour
+and refresh the last month's data.
 
-You can create multiple continuous aggregates on the same
-hypertable. For example, you can create a continuous aggregate
-`condition_summary_daily` over the data with daily buckets:
+You can now run a normal `SELECT` on the continuous aggregate and it
+will give you the aggregated data, for example, to select the averages
+for the first three months:
 
 ```sql
-CREATE VIEW conditions_summary_daily
+SELECT bucket, avg FROM conditions_summary_daily
+WHERE device = 1 AND bucket BETWEEN '2020-01-01' AND '2020-03-31'
+ORDER BY bucket;
+```
+
+### A detailed look at continuous aggregate views [](detailed-look)
+
+As you saw above, creating a refreshing [continuous
+aggregate][api-continuous-aggs] consists of creating a continuous
+aggregate view of the data using [`CREATE MATERIALIZED
+VIEW`][postgres-createview] with the `timescaledb.continuous` and
+adding a continuous aggregate policy for the continuous aggregate to
+keep it refreshed.
+
+You can create several continuous aggregates for the same
+hypertable. For example: to create a continuous aggregate view is
+updated hourly, you would write:
+
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_hourly
 WITH (timescaledb.continuous) AS
 SELECT device,
-       time_bucket(INTERVAL '1 day', time) AS bucket,
+       time_bucket(INTERVAL '1 hour', time) AS bucket,
        AVG(temperature),
        MAX(temperature),
        MIN(temperature)
 FROM conditions
 GROUP BY device, bucket;
+
+-- Create the policy
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => INTERVAL '1 month',
+	end_interval => NULL,
+	schedule_interval => INTERVAL '5 min');
 ```
 
+A `time_bucket` on the time partitioning column of the hypertable is
+required in all continuous aggregate views. If you do not provide one,
+you will get an error.
+
+When the view is created, it will (by default) be refreshed so that it
+contain an aggregate of the current data in `conditions`. If you make
+changes to the hypertable and want to refresh an interval, you can
+explicity refresh it by calling
+[`refresh_continuous_aggregate`][refresh_continuous_aggregate] and
+give the range you want to refresh:
+
+```sql
+CALL refresh_continuous_aggregate('conditions_summary_daily, '2020-05-01', '2020-06-01');
+```
+
+Note that the end range is exclusive, so this will refresh the date
```

Comment:
Need a note somewhre that all refreshes are bucket aligned. (perhaps advanced topics section?)

Reply:
I think it is more suitable for the advanced topics.

Reply:
I agree with @gayyappan. Since we might not refresh the given range, which we go into depth about here, I think it is misleading to elide the fact that we will only refresh the fully covered buckets.

I think a quick note about this is sufficient and we can go into further depth in advanced topics. For instance

"Note that the refresh will only materialize the buckets that are fully covered by the given refresh window. So, unless the window is aligned with bucket boundaries, the function will likely refresh _less_ data than specified."

Reply:
Clarified in paragraph.